docker-compose.yml

```diff
@@ -112,3 +114,15 @@ services:
       - elasticsearch
     ports:
       - "5601:5601"
+
+  # dev tools
+  profiler:
+    build:
+      context: ./docker/py-spy/
+    pid: "service:api"
+    cap_add:
+      - SYS_PTRACE
+    volumes:
+      - ./docker/py-spy/flame-graphs:/flame-graphs
+    stdin_open: true
+    tty: true
```

Comment:
Setting `stdin_open` and `tty` prevents containers from exiting when loading via Docker-Compose.